app/src/components/domain/RiskImminentEventMap/index.tsx

```diff
@@ -78,14 +98,17 @@ interface EventItemProps<EVENT> {
     onExpandClick: (eventId: number | string) => void;
 }
 
+type Footprint = GeoJSON.FeatureCollection<GeoJSON.Geometry> | undefined;
+
 interface EventDetailProps<EVENT, EXPOSURE> {
     data: EVENT;
     exposure: EXPOSURE | undefined;
     pending: boolean;
+    layers: Record<LayerType, boolean>;
+    onLayerChange: (value: boolean, name: LayerType) => void;
```

Comment:
Not sure what onLayerChange means here. Is this meant to be layer visibility?

Reply:
Yes this is the handler for layer visibility

Reply:
Let's change the prop names accordingly